lib/workspace.js

```diff
@@ -369,9 +379,38 @@ module.exports = {
       localPathExists = false;
     }
 
+    const dynamicFiles = Array.isArray(variant_params?._files)
+      ? variant_params._files
+          .map((file) => {
+            try {
+              if (!file.name) {
+                logger.warn(`Dynamic file included without a name. Ignoring file.`);
+                return null;
+              }
+              if (file.questionFile) {
+                return {
+                  name: file.name,
+                  localPath: path.join(questionBasePath, file.questionFile),
```

Comment:
I believe this is vulnerable to directory traversal, e.g. one could specify `../../../../../etc/passwd`. We should validate the `file.questionFile` is located within `questionBasePath`.

Reply:
I believe this is resolved in 4f38c71.